.github/workflows/docker-test.yml

```diff
@@ -23,11 +23,18 @@ jobs:
             const ref = context.ref
             return ref.replace(/^refs\/heads\/|-/, '')
 
+      - name: Login to GHCR
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ secrets.GHCR_USERNAME }}
+          password: ${{ secrets.GHCR_TOKEN }}
+
       - name: Create Docker Meta
         id: docker_meta
         uses: crazy-max/ghaction-docker-meta@v1
         with:
-          images: zwavejs/zwavejs2mqtt
+          images: zwavejs/zwavejs2mqtt,ghcr.io/zwavejs/zwavejs2mqtt
           tag-sha: false
```

Comment:
@onedr0p Not needed here, test images are not pushed so it is useless. Revert changes to this file

Reply:
Updated